q2_quality_filter/test/test_filter.py

```diff
@@ -16,12 +17,14 @@
 import numpy as np
 import numpy.testing as npt
 from qiime2.plugin.testing import TestPluginBase
+from qiime2.util import redirected_stdio
 from q2_types.per_sample_sequences import (
         FastqGzFormat,
-        SingleLanePerSampleSingleEndFastqDirFmt)
+        SingleLanePerSampleSingleEndFastqDirFmt,
+)
 
 from q2_quality_filter._filter import (_read_fastq_seqs, _runs_of_ones,
-                                       _truncate, q_score, q_score_joined)
+                                       _truncate)
```

Comment:
Just for the sake of consistency, I would suggest:

```python
from q2_types.per_sample_sequences import (
    FastqGzFormat,
    SingleLanePerSampleSingleEndFastqDirFmt,
)

from q2_quality_filter._filter import (
    _read_fastq_seqs,
    _runs_of_ones,
    _truncate,
)
```

8 spaces for indention at 22,23 is technically ok but probably not intentional or consistent with expectations.

Reply:
Sure, I can change that, although it isn't strictly related to this changeset.

Reply:
True.  It's up to you.  I'm not sure what's best way of handling random stuff like this.  Just something I noticed.